src/MultiselectInput.vue

```diff
@@ -1,23 +1,25 @@
 <template lang="html">
   <input
-    slot="control"
     autocapitalize="none"
     autocorrect="off"
     autocomplete="off"
     spellcheck="false"
     aria-autocomplete="list"
     type="text"
     class="multiselect__input"
+    :readonly="!searchable"
     :id="id"
-    :placeholder="computedPlaceholder"
     :value="search"
     :disabled="disabled"
     role="combobox"
-    :aria-expanded="isOpen ? 'true' : 'false'"
     aria-haspopup="true"
+    :aria-expanded="isOpen ? 'true' : 'false'"
     :aria-controls="`${id}OptionsList`"
     :aria-aria-activedescendant="`${id}${pointer}HighlightedOption`"
-    @click.stop=""
+    :aria-owns="`${id}OptionsList`"
```

Comment:
`aria-owns` is no longer needed (in lieu of `aria-controls` & `aria-expanded`) and should be dropped. This is a change in the forthcoming ARIA Authoring Practices 1.2.